docs/dev/api_usage_examples/common.py

```diff
@@ -0,0 +1,54 @@
+'''
+common.py
+'''
+
+from enum import Enum
+from time import time
+from datetime import datetime
+
+ROOT = '127.0.0.1'
+API_VERSION = 1
+TOKEN = '93fb3a42-9177-48a0-b07a-e8a7019218f6' # Replace with your dev token.
+USERNAME = 'naiveaiguy' # Replace with the username from which you got the token above.
+AUTH_HEADER = {
+    "Authorization": f"Token {TOKEN}"
+}
+
+class Track(dict):
+    """Represents a Track info dictionary.
+    """
+    def simple_repr(self):
+        """Returns a simplified representation of this Track.
+
+        It only shows the track name and the time it was listened at.
+        """
+        track_name = self["track_metadata"]["track_name"]
+        listened_at = self.get("listened_at", "Unknown")
+
+        if listened_at != "Unknown":
+            listened_at = datetime.fromtimestamp(listened_at).strftime('%Y-%m-%d %H:%M:%S')
+
+        return f"Track: {track_name}, listened at {listened_at}"
+
+EXAMPLE_RICK_TRACK = \
+    Track(listened_at=int(time()), track_metadata={
+        "additional_info": {
+            "release_mbid": "bf9e91ea-8029-4a04-a26a-224e00a83266",
+            "artist_mbids": [
+                "db92a151-1ac2-438b-bc43-b82e149ddd50"
+            ],
+            "recording_mbid": "98255a8c-017a-4bc7-8dd6-1fa36124572b",
+            "tags": ["you", "just", "got", "semi", "rick", "rolled"]
+        },
+        "artist_name": "Rick Astley",
+        "track_name": "Never Gonna Give You Up",
+        "release_name": "Whenever you need somebody"
+    })
+
+class ListenType(Enum):
+    Single = "single"
+    Playing = "playing_now"
+    Import = "import"
+
+def construct_url(resource_url):
+    return f"http://{ROOT}/{API_VERSION}/{resource_url}"
```

Comment:
Although LB runs on Python 3.6, we try to write 3.5 code as readthedocs doesn't support Python 3.6. You could use .format here, although, I would prefer explicit URLs in the `requests.post` and `requests.get` calls as they would make understanding the examples and API endpoints much easier.

Reply:
I figured it would make it easier to update for users, and for us when the API version changes. Though to be fair, if the API version changes, we'll have to change the code anyway, so I'll just remove this function and use explicit URLs.